docusaurus/docs/deployment.md

```diff
@@ -79,6 +79,14 @@ If you’re using [Apache HTTP Server](https://httpd.apache.org/), you need to c
 
 It will get copied to the `build` folder when you run `npm run build`.
 
+If you’re using [Nginx](https://www.nginx.com/), you need to modify your nginx configuration that looks like this:

```

Comment:
Check the grammar. ` that looks like this:`   `so that it looks like this:`

Reply:
> Check the grammar. ` that looks like this:` `so that it looks like this:`

Thanks! I will fix the grammar.